cms/private_media/storages.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from typing import TYPE_CHECKING
+
+from botocore.exceptions import ClientError
+from django.core.files.storage import FileSystemStorage
+from storages.backends.s3 import S3Storage
+
+if TYPE_CHECKING:
+    from django.db.models.fields.files import FieldFile
+
+
+logger = logging.getLogger(__name__)
+
+
+class PrivacySettingS3Storage(S3Storage):
+    # pylint: disable=W0223
+    private_acl_name = "private"
+    public_acl_name = "public-read"
+
+    def make_private(self, file: "FieldFile") -> bool:
+        """Make the provided file private in S3."""
+        return self._set_file_acl(file, self.private_acl_name)
+
+    def make_public(self, file: "FieldFile") -> bool:
+        """Make the provided file publically readable in S3."""
+        return self._set_file_acl(file, self.public_acl_name)
+
+    def _set_file_acl(self, file: "FieldFile", acl_name: str) -> bool:
```

Comment:
this shows as not covered by tests. It would be worth using moto with the s3 extra (we already have moto[iam]) to cover this as well

Reply:
Last comment on issue 3446 on https://github.com/getmoto/moto/issues/ has an example. Not doing the direct link as I don't want the GH cross-linking

cc @MebinAbraham what do you think?

Reply:
Added something working in [712a337](https://github.com/ONSdigital/dis-wagtail/pull/46/commits/712a3372a1cdec7c87ce40b86b9b620f154c5448) using regular mocking

Reply:
Found a way to get `mock_aws` to work, but it certainly isn't pretty: [67a4cec](https://github.com/ONSdigital/dis-wagtail/pull/46/commits/67a4ceca17af9d3d69f0a0aaf44403ac1b03dac0)